src/main/java/ro/secur/auth/configuration/ApplicationConfiguration.java

```diff
@@ -11,5 +12,10 @@
     public ModelMapper getModelMapper() {
         return new ModelMapper();
     }
+
+    @Bean
+    public ResponseFactory getResponseCreator() {
```

Comment:
also please rename this method: getResponseFactory is a better naming in our context.

Reply:
sound good